lib/plugin/instance.rb

```diff
@@ -895,7 +895,12 @@ def register_reviewable_type(reviewable_type_class)
 
   def extend_list_method(klass, method, new_attributes)
     register_name = [klass, method].join("_").underscore
-    DiscoursePluginRegistry.define_filtered_register(register_name)
+
+    # reviewable_types is already registered by DiscoursePluginRegistry
+    if register_name != "reviewable_types"
```

Comment:
Small nit but I don't think this condition is necessary since `DiscoursePluginRegistry.define_filtered_register` is a [noop](https://github.com/discourse/discourse/blob/a0e1a1216112161465f976cd1121a00f044612e3/lib/discourse_plugin_registry.rb#L41) on subsequent calls.